docs/edge-authorizers/discovery-flow.mdx

```diff
@@ -128,8 +127,8 @@ The expected response should be:
             "max_delay_seconds": 120,
             "min_delay_seconds": 60
           },
-          "resource": "<POLICY_ID>/bundle.tar.gz",
-          "service": "aserto-bundler",
+          "resource": "<TENANT>/<POLICY>/blobs/sha256:<DIGEST>",
```

Comment:
Maybe "<org-name>/<image-name>/blobs/sha256:<digest>" would be a better way to write this?  (assuming I'm interpreting correctly)

Reply:
resolver, thank you